usersimcrs/simulator/agenda_based/interaction_model.py

```diff
@@ -1,19 +1,29 @@
 """Interaction model."""
```

Comment:
Could you please briefly describe here that this model also takes care of the agenda updates?

Reply:
Done.